mote/templates/event_summary.html

```diff
@@ -2,7 +2,7 @@
   <div class="evt-smry-title d-grid d-md-block gap-2">
      <span class="head me-4">{{ meet['title'] }}</span>
      <span class="evt-smry-length text-nowrap me-4"><i class="fas fa-clock fa-lg me-2" aria-hidden="true"></i>{{ meet['duration'] }} min</span>
-     <span class="evt-smry-length text-nowrap"><i class="far fa-calendar-alt fa-lg me-2" aria-hidden="true"></i>{{ startdate }}</span>
+     <span class="evt-smry-length text-nowrap"><i class="far fa-calendar-alt fa-lg me-2" aria-hidden="true"></i>{{ startdate.strftime("%B %d, %Y") }}</span>
```

Comment:
Hi @darknao, could you please resolve the merge conflicts introduced as a result of merging #386?